app/domain/validation.py

```diff
@@ -34,6 +35,20 @@ def validate_mission(mission, submitter, for_user, creation_time=None):
     if any([not a.end_time for a in activities_to_validate]):
         raise MissionStillRunningError()
 
+    if (
+        is_admin_validation
+        and mission.submitter_id != submitter.id
+        and len(mission.validations_for(for_user)) == 0
+    ):
+        mission_start = activities_to_validate[0].start_time
+        last_activity_start = activities_to_validate[-1].start_time
+        mission_too_old = datetime.now() - mission_start > timedelta(days=10)
```

Comment:
:green_circle: on pourrait peut-etre renommer cette variable en quelque chose de positif (genre old_enough) pour etre coherent avec l'autre
activitity_too_long => error
not mission_old_enough => error plutot que 
mission_too_old => pas d'error

:orange_circle: on pourrait mettre ce fameux "10" jour au moins en constante nommee ?

Reply:
:+1: ça marche je vais faire ça !